src/task-actions.ts

```diff
@@ -339,31 +339,44 @@ async function handleContractVerification(
     const paths = [mainPath];
 
     const bodyFormData = new FormData();
-    bodyFormData.append("contract-name", path.parse(mainPath).base);
+    bodyFormData.append("compiler-version", args.compilerVersion);
+    bodyFormData.append("license", args.license || "No License (None)");
```

Comment:
If the license is invalid, what happens; what kind of message does the server return? Shouldn't we do some check here, on the client side? Or maybe log the supported versions in case of invalid input?